src/dialects/mysql/index.js

```diff
@@ -34,13 +34,15 @@ assign(Client_MySQL.prototype, {
   },
 
   QueryCompiler: QueryCompiler,
-  
+
   SchemaCompiler: SchemaCompiler,
 
   TableCompiler: TableCompiler,
 
   ColumnCompiler: ColumnCompiler,
 
+  SqlString: require('../../query/string'),
```

Comment:
This should not be needed since `src/client.js` base class already sets it to `src/query/string.js`